app/helpers/table_helper.rb

```diff
@@ -0,0 +1,42 @@
+module TableHelper
+  def sort_table_header(heading, content_item_attribute, order, sort)
+    # Determine if this column is currently sorted on
```

Comment:
I would love to find a way to get rid of lines 3-8

1. Lines 3-5 are comments, and if possible, we shouldn't add them within the code
2. Lines 5-8 could be replaced with method calls within lines 9-10

To do so, you might need to split method `next_sort` into two (one to get the label and other to get the param)

In general, the name of the methods and variable are not obvious to me. `next sort`, `current_sort`, for example, are not revealing what they return. You need to navigate to the implementation of the method to find out what they do.

What about pairing with another developer and try to find better names? Try to pair with @chao-xian for example.